tools/data-api/internal/repositories/generator_models.go

```diff
@@ -14,3 +14,64 @@ type Value struct {
 	Value       string  `json:"Value"`
 	Description *string `json:"Description,omitempty"`
 }
+
+type Model struct {
+	Fields          []Field `json:"Fields"`
+	IsDiscriminator *bool   `json:"IsDiscriminator,omitempty"`
+	ParentModelName *string `json:"ParentModelName,omitempty"`
+	ValueForType    *string `json:"ValueForType,omitempty"`
```

Comment:
This wants renaming - since that needs to be done in the other package - let's sort that package, then rely on it here.